src/Item.js

```diff
@@ -179,9 +179,13 @@ class Item extends EventEmitter {
 
     // if the item was saved with a custom inventory hydrate it
     if (this.inventory) {
+      // handle equipped containers.
+      if (!(this.inventory instanceof Inventory)) {
```

Comment:
If you'd like I can remove this from the PR and test it out to make sure that it's not just an issue on my part.